packages/@okta/vuepress-site/docs/reference/rl-system-log-events/index.md

```diff
@@ -0,0 +1,267 @@
+---
+title: System Log Event Guidance for rate limits
+category: rate limits
+---
+
+# System Log event guidance for rate limits
+
+Okta supports a few standard System Log events for your use with rate limiting. If you are implementing anything that can reject an operation within Okta because it has exceeded a [rate limit](/docs/reference/rate-limits/) (attempts per unit time), then use these standard System Log event types.
+
+The HTML response code returned when rejected should be HTTP 429.
+
+## System Log event types for rate limits
+
+The following System Log events for use with rate limiting are available:
+
+* `system.operation.rate_limit.violation`<br>
+Emit this event type once per rate limiting period when rejecting a request for exceeding a rate limit. For example, if a client exceeds their calls per minute quota, then you should emit one event within that window maximum.
+
+* `system.operation.rate_limit.warning`<br>
+If you want to warn the caller that some significant portion of their rate limit has already been used within a period, you may also emit this event type once per rate limiting period. For example, you warn the customer that 60% of their daily limit for event delivery has been reached.
+
+* `system.operation.rate_limit.notification`<br>
+If you want to provide additional information about rate limiting decisions you may emit this event type. For example, Elastic Rate Limiting (ERL) was activated for a customer within a rate limiting period, or a violation event would have been emitted if the customer had a different configuration.
+
+## DebugContext object
+
+For some types of events, the fields provided in other response objects aren't sufficient to adequately describe the operations that the event has performed. In such cases, the [DebugContext](/docs/reference/api/system-log/#debugcontext-object) object provides a way to store additional information.
+
+## DebugContext object properties for rate limiting
+
+The following table describes the rate limit information that is returned in the DebugContext object.
+
+> **Important:** The information contained in `debugContext.debugData` is intended to add context when troubleshooting customer platform issues. Note that both key names and values may change from release to release and aren't guaranteed to be stable. Therefore, they shouldn't be viewed as a data contract but as a debugging aid instead.
+
+| Property                           | Type                     | Description                                                                                                       |
+| ---------------------------------- | ------------------------ | ----------------------------------------------------------------------------------------------------------------- |
+| `operationRateLimitScopeType`      | ??what is the datatype?? | The type of rate limit scope effected. Example scopes: `org`, `user`, `application`                               |
```

Comment:
These are all String type

Reply:
@peteryaworsky-okta do we want to have application listed here?